### PR TITLE
Image support

### PR DESCRIPTION
**Pull request by vasiliybondarenko**
This PR implements images support (issue #48 ). 
I've modified Getting Started example to show how it works. If it looks good I can proceed and add the appropriate unit tests.

### CODE REVIEW

**Comment by norbert-radyk** [comment]
Hi Vasily, the changes look good and are definitely worth adding to spoiwo. As you've mentioned some tests are missing so please add them to the PR so we can keep up the good coverage. 

Also might be worth adding  the matching examples in Java and Scala to https://github.com/norbert-radyk/spoiwo/tree/master/examples/src/main/scala/com/norbitltd/spoiwo/examples/quickguide based on http://poi.apache.org/components/spreadsheet/quick-guide.html#Images rather than modifying the getting started example - as inserting an image is not the first thing you want to do when you open/create a spreadsheet :)

File: src/main/scala/com/norbitltd/spoiwo/model/Image.scala [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.norbitltd.spoiwo.model
+
+/**
```
Comment (norbert-radyk) [inline src/main/scala/com/norbitltd/spoiwo/model/Image.scala]: The rest of the codebase doesn't have a scaladoc comments and I would suggest to keep it that way as they tend to go out of date quickly and replicate the snapshot of information readily available in git. Obviously if you really want to keep it to have your name in codebase I won't see it as a showstopper, given the code looks good.

Comment (vasiliybondarenko) [reply src/main/scala/com/norbitltd/spoiwo/model/Image.scala]: Thanks the feedback! I'll fix comments and add examples/tests.

Comment (vasiliybondarenko) [reply src/main/scala/com/norbitltd/spoiwo/model/Image.scala]: Just added tests and example.

**Pull request closed without merging**